server/tests/queries/getServices.js

```diff
@@ -0,0 +1,12 @@
+const test = require('tape');
+
+const { getServices } = require('../../db/queries/getServices');
+
+test('testing for getServices query database', async (t) => {
+  const obj = await getServices();
```

Comment:
```suggestion
  const services = await getServices();
```
and change it in the rest of the file , use descriptive names